src/user-event/paste.js

```diff
@@ -0,0 +1,120 @@
+import {getConfig as getDOMTestingLibraryConfig} from '../config'
+import {fireEvent} from '../events'
```

Comment:
Make sure to use the fireEvent from the utils module because it's async which is required for user event methods.